packages/truffle-db/src/workspace/test/bytecode.spec.ts

```diff
@@ -0,0 +1,67 @@
+import gql from "graphql-tag";
+import { generateId, Migrations, WorkspaceClient } from './utils';
+
+describe("Bytecode", () => {
+  let wsClient;
+  let expectedId;
+  let variables, addBytecodeResult;
+
+  beforeEach(async () => {
+    wsClient = new WorkspaceClient();
+    expectedId = generateId({ bytes: Migrations.bytecode });
+    variables = { bytes: Migrations.bytecode };
+    addBytecodeResult = await wsClient.execute(AddBytecode, { bytes: variables.bytes });
+  });
+
+  test ("can be added", async () => {
+    expect(addBytecodeResult).toHaveProperty("bytecodesAdd");
+
+    const { bytecodesAdd } = addBytecodeResult;
+    expect(bytecodesAdd).toHaveProperty("bytecodes");
+
+    const { bytecodes } = bytecodesAdd;
+    expect(bytecodes).toHaveLength(1);
+
+    const bytecode = bytecodes[0];
+    expect(bytecode).toHaveProperty("id");
+
+    const { id } = bytecode;
+    expect(id).toEqual(expectedId);
+  });
+
+  test("can be queried", async() =>{
+    const executionResult = await wsClient.execute(GetBytecode, { id: expectedId });
+    expect(executionResult).toHaveProperty("bytecode");
+
+    const { bytecode } = executionResult;
+    expect(bytecode).toHaveProperty("id");
+    expect(bytecode).toHaveProperty("bytes");
+
+    const { id, bytes } = bytecode;
+    expect(id).toEqual(expectedId);
+    expect(bytes).toEqual(variables.bytes);
+  });
+});
+
+export const GetBytecode = gql`
+  query GetBytecode($id: ID!) {
+    bytecode(id: $id) {
+      id
+      bytes
+    }
+  }
+`
+
+export const AddBytecode = gql`
+  mutation AddBytecode($bytes: Bytes!) {
+    bytecodesAdd(input: {
+      bytecodes: [{
+        bytes: $bytes
+      }]
+    }) {
+      bytecodes {
+        id
+      }
+    }
+  }
+`
```

Comment:
need new line

Reply:
This is strange.  `truffle-db` hasn't merged in the lint-staged config setup for TS files from `develop`. My expectation is that it would be handled by that, though I'll test to verify.